histolab/tile.py

```diff
@@ -168,7 +168,22 @@ def save(self, path: Union[str, bytes, os.PathLike]) -> None:
 
     @lazyproperty
     def tissue_mask(self) -> np.ndarray:
-        """Binary mask representing the tissue in the tile.
+        """Binary mask representing the tissue in the tile calculated
```

Comment:
please keep the first line and the body separated with a blank line, and, as before, please follow the PEP 257 rules 🙏🏽 . Pay attention : The summary line must fit in one line (88chars)